src/js/modules/rpc/server.ts

```diff
@@ -194,7 +194,8 @@ export class ProcessBatchServer extends SupervisorServer {
      * from a string, the first and second arguments are the parameter for
      * that function and the last one is the js string program.
      */
-    const loadScript = Function("module", "require", script.toString());
+    const loadScript = (module, requireNative) =>
+      Function("module", "require", script.toString())(module, requireNative);
```

Comment:
shouldn't this be in a try/catch?

Reply:
no I use the try and catch in place that I called `loadScript`